app/client/cli/peer/connections.go

```diff
@@ -0,0 +1,80 @@
+package peer
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/pokt-network/pocket/app/client/cli/helpers"
+	"github.com/pokt-network/pocket/p2p/debug"
+	"github.com/pokt-network/pocket/shared/messaging"
+)
+
+var (
+	connectionsCmd = &cobra.Command{
```

Comment:
Everywhere else we have a function that returns a new instance of `cobra.Command`.

I think we should either:
1. Stay consisstent
2. Stay consistent + add a TODO to change the others
3. Change all of them